models/meshmodel/core/v1beta1/application_component.go

```diff
@@ -1,22 +1,22 @@
-package v1alpha1
+package v1beta1
 
 import (
 	"fmt"
 	"strings"
 
-	"github.com/layer5io/meshkit/models/meshmodel/core/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
+// To be removed when Design schema is refactored.
 
-// Component is the structure for the core OAM Application Component
+// Component is the structure for the core Application Component
 type Component struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
 	Spec ComponentSpec `json:"spec,omitempty"`
 }
 
-// ComponentSpec is the structure for the core OAM Application Component Spec
+// ComponentSpec is the structure for the core Application Component Spec
```

Comment:
Clean this up, please.

Reply:
File needs renamed?

What is `metav1`? Is there a schema for this?

Reply:
No, the namings have been inspired from k8s packages, v1 in metav1 eflects the schema version.


These are OAM constructs that is most appropriate to address with v1beta1 version of design schema